### ISSUE TITLE
where to modify for Heroku ephemeral file system?

### ISSUE THREAD

**Issue by jasonfb** [dz0ny/heroku-docker-piwik#13]
changes to database settings appear not to persist when Heroku instance is restarted

I have also attempted to hard code DB settings into config.ini.php but this does not persist in deployed app either.

**Comment by jasonfb** [comment]
Upon restarting app, I am consistently returned to this screen
![2017-03-28_2012](https://cloud.githubusercontent.com/assets/59002/24432887/0573dd28-13f3-11e7-9063-5a4ba549c0b4.png)

**Comment by jasonfb** [comment]
in addition to having Heroku config vars DB_HOST, DB_NAME, DB_PASSWORD, DB_PORT, DB_PREFIX, and DB_USERNAME set, I also have DATABASE_URL set, but none of these seem to persist or get picked up

**Comment by jasonfb** [comment]
After much trial & error I've concluded that Heroku is simply not right for Piwik. Piwik is very specialized and highly reliant on writing back to the file system. Trying to debug the Heroku docker build I also found very difficult, and was never able to correctly configure the installation in such a way that the settings in ```config.ini.php``` would actually persist (they would always be wiped away when the app restarted)

Instead, I attempted to spin up a simple EC2 instance on Amazon, installed PHP & nginx in basically no time flat — I had a few hiccups involving mbstring and some other PHP dependancies —  and then Piwik "just worked" out of the box the way I was expecting it to. Spinning it up on Amazon took me about 1 hour. Fighting with the Heroku settings and configuration took me several days and lead nowhere. 

I am concluding that Heroku is the wrong platform for Piwik
